.github/workflows/sync-docs-from-node.yml

```
@@ -94,7 +93,18 @@ jobs:
       - name: Create branch for changes
         run: |
           set -euo pipefail
-          BRANCH_NAME="docs/node/${{ steps.params.outputs.tag }}"
+          # Sanitize version string for use in branch name
+          VERSION="${{ steps.params.outputs.version }}"
+          SAFE_VERSION=$(echo "$VERSION" | sed 's/\//-/g')  # replace any '/' with '-'
+          BRANCH_NAME="docs/node/${SAFE_VERSION}"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)